src/twisted/conch/test/test_cftp.py

```diff
@@ -46,7 +53,9 @@
 
 
 
-
+@skipIf(None in (unix, cryptography, pyasn1,
+                 interfaces.IReactorProcess(reactor, None)),
+        "don't run w/o spawnProcess or cryptography or pyasn1")
```

Comment:
I see that the condition is repeated... maybe keep the similar condition in a variable

```suggestion
cftp_skip_conditions = (
    None in (unix, cryptography, pyasn1, interfaces.IReactorProcess(reactor, None)),
    "don't run w/o spawnProcess or cryptography or pyasn1",
    )
@skipIf(*cftp_skip_conditions)
```

Reply:
OK I changed this condition by using a variable.